src/main/java/duke/Event.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+
+public class Event extends Task {
+    // protected String from;
+    // protected String to;
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
```

Comment:
nice to see that you went beyond what was asked in the iP and made Event have LocalDateTime fields as well. However, this causes some user experience problems, because now in order to enter a Event task you have to write a whole big chunk of text 😬